index.html

```diff
@@ -325,6 +326,9 @@
         [[!WEBIDL]].
       </p>
       <p>
+        The interface <dfn>PresentationRequest</dfn> is defined in [[PRESENTATION-API]].
```

Comment:
Editorial suggestion: wrap this interface definition in `<code>` block:

`<dfn><code>PresentationRequest</code></dfn>`

Now the `<code>` style is inherited elsewhere when referenced with just `<a>PresentationRequest</a>`.

Reply:
Done.